models/Model.js

```diff
@@ -41,10 +41,31 @@ selectArticleComments = (article_id) => {
     });
 };
 
+postTheCommentTo = (responseBody, article_id) => {
+  const { username, body } = responseBody;
+  return fs
+    .readFile(`${__dirname}/../db/data/test-data/users.js`, "utf-8")
```

Comment:
You should not be reading the test data file - when you host your application it will be using production data so this won't work. 

If the username does not exist in the database psql will throw an error, catch it in the controller and send it to your error handlers - it is not necessary to check that the username exists